modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,396 @@
+----
+
+.. Identify the
+ifndef::openshift-rosa[]
+{product-title}
+endif::openshift-rosa[]
+ifdef::openshift-rosa[]
+ROSA
+endif::openshift-rosa[]
+cluster OIDC DNS:
++
+--
+[source, terminal]
+----
+$ rosa describe cluster --cluster=<cluster_name> | grep -i 'OIDC'
+----
+
+Save the output from both commands. You will use this information in future steps within this procedure.
+
+--
+. Create the AWS IAM policy required for the ALBO:
++
+.. Create ALBO namespace:
++
+[source,terminal]
+----
+$ oc create namespace aws-load-balancer-operator
+----
++
+.. Assign the following trust policy to the newly-created AWS IAM role. For more information on assigning trust policies to AWS IAM roles, see link:https://aws.amazon.com/blogs/security/how-to-use-trust-policies-with-iam-roles/[How to use trust policies with IAM roles].
++
+[source,json]
+----
+IDP='{Cluster_OIDC_Endpoint}'
+IDP_ARN="arn:aws:iam::<my-aws-account>:oidc-provider/${IDP}" <1>
+cat <<EOF > albo-operator-trusted-policy.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Federated": "${IDP_ARN}"
+            },
+            "Action": "sts:AssumeRoleWithWebIdentity",
+            "Condition": {
+                "StringEquals": {
+                    "${IDP}:sub": "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-operator-controller-manager"
+                }
+            }
+        }
+    ]
+}
+EOF
+----
+<1> Replace '{AWS_AccountNo}' with your AWS account number and '{Cluster_OIDC_Endpoint}' with the OIDC DNS identified earlier in this procedure.
++
+[NOTE]
+====
+Do not include the `https` portion of the OIDC DNS URL when replacing `{Cluster_OIDC_Endpoint}` with the OIDC DNS you identified earlier. Only the alphanumeric information that follows the `/` within the URL is needed.
+====
++
+.. Create and verify the role with the generated trusted policy:
++
+[source, terminal]
+----
+$ aws iam create-role --role-name albo-operator --assume-role-policy-document file://albo-operator-trusted-policy.json
+OPERATOR_ROLE_ARN=$(aws iam get-role --role-name albo-operator | \grep '^ROLE' | \grep -Po 'arn:aws:iam[0-9a-z/:\-_]+')
+echo $OPERATOR_ROLE_ARN
+----
++
+.. Attach the operator's permission policy to the role:
++
+[source, terminal]
+----
+$ curl -o albo-operator-permission-policy.json https://raw.githubusercontent.com/alebedev87/aws-load-balancer-operator/aws-cli-commands-for-sts/hack/operator-permission-policy.json
+aws iam put-role-policy --role-name albo-operator --policy-name perms-policy-albo-operator --policy-document file://albo-operator-permission-policy.json
+----
++
+. Generate the controller's AWS credentials:
++
+[source,terminal]
+----
+$ cat <<EOF> albo-operator-aws-credentials.cfg
+[default]
+sts_regional_endpoints = regional
+role_arn = ${OPERATOR_ROLE_ARN}
+web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+EOF
+----
++
+//[NOTE]
```

Comment:
I know this is commented out, but I recommend changing this NOTE to IMPORTANT as well.